security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -57,8 +57,9 @@ const (
 )
 
 var (
-	sdsClients      = map[cache.ConnKey]*sdsConnection{}
-	sdsClientsMutex sync.RWMutex
+	sdsClients       = map[cache.ConnKey]*sdsConnection{}
```

Comment:
[optional] May be have a struct for all these, and provide get/set functions that always use mutex

Reply:
I am have a working PR for these.

Reply:
sync map should just work?